tests/flow/test_compatibility_contract.py

```
@@ -0,0 +1,262 @@
+            "Procedure `db.noSuchProc` is not registered",
+        )
+
+    def test08_indexes_catalog_contract(self):
+        graph_name = "compat_indexes"
+        self.conn.delete(graph_name)
+
+        self._query(graph_name, "CALL db.idx.fulltext.createNodeIndex('Person', 'name')")
+        result = None
+        for _ in range(20):
+            result = self._query(graph_name, "CALL db.indexes()")
+            if result[1] and dict(zip(result[0], result[1][0]))["status"] == "OPERATIONAL":
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)